url-fixer/index.js

```diff
@@ -168,4 +168,10 @@ async function main(){
   await generateCsv(links)
 }
 
-main()
+module.exports = {
+  readFile,
+  writeFile,
+  findAndReplaceAll
+}
+
+if (process.main === module) main()
```

Comment:
Borrowing some functions from the script used for fixing redirects that came out of an SEO audit.